server/service/scenarios/index.js

```diff
@@ -23,4 +23,9 @@ scenariosRouter.post('/:scenario_id', [
 
 scenariosRouter.delete('/:scenario_id', [lookupScenario(), deleteScenario]);
 
+scenariosRouter.use('/:scenario_id/slides', [
+    lookupScenario(),
+    require('./slides')
```

Comment:
I'm wondering if I understand this right: we use the `require('./slides')` middleware statement in the route itself so it only loads if we hit that route? And we don't require it at the top of the page since we don't always need it for the other routes and can probably avoid loading it?